test/integration/targets/listen_ports_facts/aliases

```diff
@@ -0,0 +1,6 @@
+shippable/posix/group1
+posix/ci/group2
```

Comment:
This alias is obsolete (replaced by the `shippable/posix/...` aliases), so it should be removed.